packages/flutter_tools/lib/src/runner/target_devices.dart

```diff
@@ -302,3 +338,397 @@ class TargetDevices {
     return result;
   }
 }
+
+@visibleForTesting
+class MacOSTargetDevices extends TargetDevices {
```

Comment:
This name seems like it's deploying a macOS desktop app?

Reply:
I changed it to `TargetDevicesWithExtendedWirelessDeviceDiscovery`. I changed the one in the devices command too to `DevicesCommandOutputWithExtendedWirelessDeviceDiscovery`. Let me know if you think that's better